groups.go

```diff
@@ -98,6 +98,7 @@ type CreateGroupOptions struct {
 	Name        string `url:"name,omitempty" json:"name,omitempty"`
 	Path        string `url:"path,omitempty" json:"path,omitempty"`
 	Description string `url:"description,omitempty" json:"description,omitempty"`
+	Visibility  int    `url:"visibility_level" json:"visibility_level"`
```

Comment:
There is a type for the visibility level that you should use instead: https://github.com/xanzy/go-gitlab/blob/master/gitlab.go#L87

Also in other places this is called `VisibilityLevel` instead of just `Visibility`. If you can change both these things we should be good to go!